jiant/tasks/lib/panx.py

```diff
@@ -34,9 +34,12 @@ def tokenize(self, tokenizer):
         for token, pos in zip_equal(self.tokens, self.pos_list):
             # Tokenize each "token" separately, assign label only to first token
             tokenized = tokenizer.tokenize(token)
+            # If the token can't be tokenized, or is too long, replace with a single <unk>
+            if len(tokenized) == 0 or len(tokenized) > 100:
```

Comment:
Where does the requirement of 100 come from? Can this be changed to a constant with an explanation?

Reply:
Updated.